components/Branding/LogoSection/LogoSection.js

```diff
@@ -15,40 +16,31 @@ function LogoSection() {
         <ul className={styles.logoSizeList}>
           {/* - SMALL LOGOS - */}
           <li className={classNames(styles.logoSizeListItem, styles.smallLogos)}>
-            <h5>Small Logos</h5>
+            <h5>Logos</h5>
             <p>
-              For use when Operation Code&apos;s logo name is between 0-1 inch in height. In most
-              cases, use the Original Small Logo. The Stacked Small Logo is to be used where
-              graphics needs are larger in vertical height than horizontal width with the Operation
-              Code logo name still under 1 inch in height.
+              {/* eslint-disable-next-line max-len */}
+              Operation Code&apos;s logo is provided as code itself. It is an SVG grouped into four
+              color fill areas. SVG is appropriate to use for small and large formats where
+              horizontal space is needed.
             </p>
 
             <ul className={styles.logoTypeList}>
               <li className={styles.logoTypeListItem}>
                 <h6>TYPICAL</h6>
                 <ul className={styles.badgeList}>
                   <li>
-                    <Badge
-                      icon={
-                        <Image
-                          src={`${s3}branding/logos/small-blue-logo.png`}
```

Comment:
I liked changing our nav's logo to be an SVG, and I think it's cool that you made a logo component; however, I think that it's important to continue offering the previous `.png` files for people who have no idea what to do with an SVG. Imagine - for example - somebody doing a news story on us. They'll be super excited that we offer a brand page for them, but they may use a CMS that doesn't support SVGs for their image upload.

Reply:
1. Convenience factor (that's the whole reason this page exists)
2. Because then there'd _definitely_ be quality loss which is the problem you're attempting to solve with using SVGs.

Reply:
We'll just do this on the front end so we don't have to maintain png files too